app/models/ab_test_assignment.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AbTestAssignment < ApplicationRecord
+  class << self
+    def bucket(**)
+      find_by(**)&.bucket&.to_sym
```

Comment:
```suggestion
      find_by(**)&.bucket&.to_sym || :default
```
do we want a default name or nil is fine?

Reply:
`nil` and `:default` are meaningfully different in this context. `nil` would represent the absence of an assignment ([source](https://github.com/18F/identity-idp/blob/54c4274264bba552e2c214b3d720e08bcb8ec9d7/lib/ab_test.rb#L86)). The [default bucket will be persisted](https://github.com/18F/identity-idp/blob/54c4274264bba552e2c214b3d720e08bcb8ec9d7/lib/ab_test.rb#L92) and returned by `AbTest#bucket` if chosen. A `nil` return value from `bucket` is expected to be [handled as though it's inactive](https://github.com/18F/identity-idp/blob/54c4274264bba552e2c214b3d720e08bcb8ec9d7/lib/ab_test.rb#L77).